modules/tinymce/src/themes/silver/main/ts/ui/core/complex/BespokeSelect.ts

```diff
@@ -110,7 +110,7 @@ const generateSelectItems = (_editor: Editor, backstage: UiFactoryBackstage, spe
         text: translatedText,
         icon: rawItem.icon,
         active: rawItem.isSelected(value),
-        disabled,
+        enabled: !disabled,
```

Comment:
This variable was probably poorly named previously... can we please rename this to `invalid` as that's what it's meant to represent by the looks?

Reply:
@metricjs ?

Reply:
I replaced `disabled` argument with `invalid` as requested